pkg/pool-weighted/contracts/managed/ManagedPool.sol

```diff
@@ -28,27 +28,28 @@ import "./ManagedPoolSettings.sol";
 
 /**
  * @title Managed Pool
- * @dev Weighted Pool with mutable tokens and weights, designed to be used in conjunction with a pool controller
- * contract (as the owner, containing any specific business logic). Since the pool itself permits "dangerous"
- * operations, it should never be deployed with an EOA as the owner.
+ * @dev Weighted Pool with mutable tokens and weights, designed to be used in conjunction with a contract
+ * sometimes called a controller (as the owner, containing any specific business logic). Since the pool
+ * itself permits "dangerous" operations, it should never be deployed with an EOA as the owner.
  *
- * Pool controllers can add functionality: for example, allow the effective "owner" to be transferred to another
- * address. (The actual pool owner is still immutable, set to the pool controller contract.) Another pool owner
- * might allow fine-grained permissioning of protected operations: perhaps a multisig can add/remove tokens, but
- * a third-party EOA is allowed to set the swap fees.
+ * Pool owners can add functionality: for example, allow the effective "owner" (sometimes called a manager),
+ * to be transferred to another address. (The underlying pool owner is still immutable, set to the address
+ * of the contract it was deployed with.) Another pool owner might allow fine-grained permissioning of
+ * protected operations: perhaps a multisig can add/remove tokens, but a third-party EOA is allowed to set
+ * the swap fees.
```

Comment:
```suggestion
 * The owner contract can impose arbitrary access control schemes on its permissions: it might allow a multisig to add
 * or remove tokens, and let an EOA set the swap fees.
```

I wouldn't go into too much detail on the owner contract while describing the pool itself - it might be a bit confusing.